build.sbt

```diff
@@ -1,10 +1,10 @@
-lazy val finagleVersion = "18.4.0"
+lazy val finagleVersion = "18.6.0"
```

Comment:
why not bump this all the way up to 18.7.0?

Reply:
Same question. The most recent Finch release is against 18.7.